recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py2k]
+  skip: true  # [py2k or py<=36]
```

Comment:
Python 2.x causes `py<=36` to evaluate to True, so the `py2k` clause is no longer necesasry.